language/anope.de_DE.po

```diff
@@ -1924,6 +1907,9 @@ msgstr "Zugriffs-Liste von %s:"
 msgid "Access to this command requires the permission %s to be present in your opertype."
 msgstr "Um Zugriff auf diesen Befehl zu haben benötigst Du die Berechtigung %s in Deinem Oper Type."
 
+msgid "Account"
+msgstr ""
```

Comment:
Translation missing here.

Reply:
FWIW: update.sh added these lines, AFAICT

Reply:
Yeah that's fine. It's a new translation string added since the translation file was last updated.